blog/_posts/2020-05-26-create-your-own-theme-detailed-walkthrough.md

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Create your own theme in minutes !
+---
+
+In Convos [v4.xx](https://convos.chat/blog/2020/5/14/theming-support-in-4-point-oh.html) it is super easy to customize
+or create a theme.  Follow this detailed walkthrough where I create a **"hack3r"** theme ! 
```

Comment:
We can't have "I", since it's written by "you" :) Can we have something like this instead:

> In Convos \[v4.xx]\(https://convos.chat/blog/2020/5/14/theming-support-in-4-point-oh.html)
> it is super easy to customize or create a theme. Follow this detailed
> walkthrough written by \[Thibault Duponchelle]\(https://whatever.link.you.want)
> on how to create a \*\*"hack3r"** theme!

Reply:
Ok, changed like you proposed, I had no idea what to link then I linked my github